### PR TITLE
Fix: show correct number of decimals in content manager

### PR DESCRIPTION
**Pull request by oscarsandberg**
### What does it do?

Fixes a UI bug in the table in content manager. Now all decimals for values with the type float or decimal is shown.

### Why is it needed?

Previously values with the type float or decimal were rounded to 3 decimals if the value contained more than 3 decimals.

### How to test it?

1. Add a new number field
2. Set to float (ex: 3.33333333) or decimal (ex: 2.22)
3. Go to content editor and add a floating point number (example, lat/long) or decimal with more than 3 decimals.
4. Save.
4. The value in the table is not rounded to 3 decimal

### Related issue(s)/PR(s)

Solves part of the issue related issue: #11893

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js [hunk 1]
```diff
@@ -27,13 +27,31 @@ const CellValue = ({ type, value }) => {
...
+    formattedValue = formatNumber(value, { maximumFractionDigits: 0 });
   }
 
   return toString(formattedValue);
 };
 
+const getNumberOfDecimals = value => {
+  if (value % 1 !== 0) {
+    // value has decimals
+    return value.toString().split('.')[1].length;
```
Comment (gu-stav) [inline packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js:49]: Unfortunately this doesn't apply to every language. In german e.g. the decimal seperator is ",". The design-system has already solved this by using `@internationalized/number`.

https://github.com/strapi/design-system/blob/develop/packages/strapi-design-system/src/NumberInput/NumberInput.js#L6

Would it make sense to unify this behavoir?

Comment (oscarsandberg) [reply packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js:49]: I see that I first misunderstood the bug. This PR solves the problem of values rounding in the table in content manager, but not the click outside in input field

File: packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js [hunk 2]
```diff
@@ -27,13 +27,31 @@ const CellValue = ({ type, value }) => {
...
+  }
+
+  if (['integer', 'biginteger'].includes(type)) {
+    formattedValue = formatNumber(value, { maximumFractionDigits: 0 });
   }
 
   return toString(formattedValue);
 };
 
+const getNumberOfDecimals = value => {
```
Comment (HichamELBSI) [inline packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js:46]: Hi @oscarsandberg, thanks for your contribution. 
Can you move this function in a utils file please : `CellContent/utils/getNumberOfDecimals.js` and add some test in `CellContent/utils/tests/getNumberOfDecimals.test.js` please 😄 .

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js (hunk 2 above)
Comment (oscarsandberg) [reply packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js]: Hi @HichamELBSI, thanks for the comments. I have moved the function and added som tests. Let me know if you want me to do anything else :)

**Review by HichamELBSI — APPROVED** [review]
LGTM thanks 😄  Can you update the branch so we can merge please.

**Pull request merged**